sway/tree/output.c

```diff
@@ -300,13 +297,6 @@ void output_disable(struct sway_output *output) {
 	list_del(root->outputs, index);
 
 	output->enabled = false;
-
-	arrange_root();
-
-	// Reconfigure all devices, since devices with map_to_output directives for
-	// an output that goes offline should stop sending events as long as the
-	// output remains offline.
-	input_manager_configure_all_input_mappings();
```

Comment:
This function isn't called in the new code path. This function may not be necessary.

Reply:
It's already present in the modeset codepath so it doesn't show on first glance in the diff: https://github.com/swaywm/sway/pull/8326/files#diff-66d6f9399c138857218e684475e604f3710684dd9deb896b66bf0ec2cacd149fR987